catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java

```diff
@@ -295,18 +301,46 @@ private void showProgressDialogAndUploadProject() {
 
 		uploadProgressDialog.show();
 		uploadProgressDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(false);
+	}
 
-		Intent intent = new Intent(this, ProjectUploadService.class);
-		String[] sceneNames = project.getSceneNames().toArray(new String[0]);
-		intent.putExtra(EXTRA_RESULT_RECEIVER, uploadResultReceiver);
+	public Project getProject() {
```

Comment:
please inline this

Reply:
I get checkstyle errors when i inline this.

Reply:
I tried this and get no checkstyle errors. Remove the `getProject` function and use `project` directly.
```
diff --git a/catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java b/catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java
index da1c836f5c..35d5ddb1c9 100644
--- a/catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java
+++ b/catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java
@@ -257,7 +257,7 @@ public class ProjectUploadActivity extends BaseActivity implements
 				})
 				.setPositiveButton(getText(R.string.next), (dialog, which) -> {
 					project.setTags(checkedTags);
-					projectUploadController.startUpload(getProjectName(), getProjectDescription(), getProject());
+					projectUploadController.startUpload(getProjectName(), getProjectDescription(), project);
 				})
 				.setNegativeButton(getText(R.string.cancel), (dialog, which) -> {
 					Utils.invalidateLoginTokenIfUserRestricted(this);
@@ -303,8 +303,6 @@ public class ProjectUploadActivity extends BaseActivity implements
 		uploadProgressDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(false);
 	}
 
-	public Project getProject() { return project; }
-
 	@Override
 	public ResultReceiverWrapper getResultReceiverWrapper() {
 		return uploadResultReceiver;
```